no-computer.md

```diff
@@ -46,6 +82,13 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I did once see a friend make a turing machine to describe the different moves
+he could switch between. Otherwise, I don't know of any other West Coast Swing
+DSLS, as there is no subset to WCS. However, one could argue that social 
+dancing in general is a DSL, and that WCS is a subset of that. Examples of
+other social dances include Lindy (https://www.youtube.com/watch?v=v9xxeWRxSbA), 
+blues (https://www.youtube.com/watch?v=-m0V2Zq-Qf4), and Salsa
+(https://www.youtube.com/watch?v=3nd8WQy58Pw).
 
```

Comment:
With your definition of what a program is and how it runs, its seems like there is a commonly used DSL, the one that is used in most classes to tell the students what to do and then have them execute the program by attempting the dance.